locales/en-US/messages.properties

```diff
@@ -68,6 +68,86 @@ navViewTutorial=TUTORIAL
 navToggleAutoUpdateTitle=Toggle Preview Auto-refresh
 navToggleAutoUpdateLabel=Auto
 
+# Code Snippet Data
+snippetHTMLComment=Comment
+snippetHTMLCommentTitle=Add a Comment
+snippetHTMLCommentData=<!-- Your HTML comment here -->\n
```

Comment:
no makes perfect sense! Thanks for the input! I think I'll go with that approach.